wire-ios-mocktransport/Tests/Source/Search/MockTransportSessionAddressBookTests.m

```diff
@@ -37,77 +37,6 @@ - (void)testThatWhenPostingAddressBookItReturnsAllUserIDsThatAreNotSelf
         user2 = [session insertUserWithName:@"User2 AABB"];
     }];
     WaitForAllGroupsToBeEmpty(0.5);
-    
-    
-    // WHEN
-    NSDictionary *upstreamPayload = @{
-                                      @"self":@[@"2312rfw32434234"],
-                                      @"cards": @[@{@"contact":@[]}]
-                                      };
-    NSString *path = [NSString pathWithComponents:@[@"/", @"onboarding", @"v3"]];
```

Comment:
`/onboarding` has been removed in v8.
Also I noticed that we don't use the endpoint, so I removed it from `WireMockTransport`.